### PR TITLE
Add docker-compose.yaml and change Makefile

### PR DESCRIPTION
**Pull request by DominikSerwatka**
I think now it should be ok

### CODE REVIEW

**Comment by jdabrowski** [comment]
on the bottom of the page you can see that there is a conflict with `dev` branch
resolve it locally:
```
git checkout dev
git pull
git checkout featurePostgres
git merge dev
<resolve conflicts in Intellij>
git commit (with default message)
ggp
```

**Comment by jdabrowski** [comment]
Also I would suggest to extend config a little to have: explicit version, health check and logs:
```
container_name: postgres
    image: postgres:14.2
    restart: on-failure
    command: [ "postgres", "-c", "log_statement=all" ]
    networks:
      - system
    healthcheck:
      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
      interval: 10s
      timeout: 5s
      retries: 5
    ports:
      - 5432:5432
    environment:
      - POSTGRES_PASSWORD=password
      - POSTGRES_USER=user
      - POSTGRES_DB=sharedparkingspot
    logging:
      driver: "json-file"
      options:
        max-size: 50m
```

**Comment by jdabrowski** [comment]
to connect to DB I recommend pgcli:
```
➜  ~ pgcli "postgres://user:password@localhost/sharedparkingspot?sslmode=disable"
Server: PostgreSQL 14.2 (Debian 14.2-1.pgdg110+1)
Version: 3.5.0
Home: http://pgcli.com
sharedparkingspot>
```

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+	bash wait.sh
 run:
-	cd shared-parking-spot-app/ && \
-    mvn clean install && \
-    cd .. && \
-    docker build -t shared-parking-spot . && \
-    docker run -d -p 7777:7777 shared-parking-spot
+	mvn clean install -DskipTests
+	docker build -t shared-parking-spot .
+	docker run -d -p 7777:7777 --network=shared-parking-spot_system --name app shared-parking-spot
```
Comment (jdabrowski) [inline Makefile]: method `run` should not have -d as we want to be in attach mode

File: shared-parking-spot-app/src/main/resources/application.yaml [hunk 2]
```diff
@@ -1,2 +1,20 @@
 server:
-  port: 7777
\ No newline at end of file
+  port: 7777
+spring:
```
Comment (jdabrowski) [inline shared-parking-spot-app/src/main/resources/application.yaml]: I think it's not necessary now

**Comment by jdabrowski** [comment]
good enough - my PR (https://github.com/parking-sharing/shared-parking-spot/pull/10) will fix some minor issues

**Review by jdabrowski — APPROVED** [review]

**Review by ZbigniewStasienko — APPROVED** [review]

**Pull request merged**